cpp/daal/include/algorithms/gradient_boosted_trees/gbt_classification_model_builder.h

```diff
@@ -123,10 +123,10 @@ class DAAL_EXPORT ModelBuilder
     *  \param[in] featureValue    Feature value for splitting
     *  \return Node identifier
     */
-    NodeId addSplitNode(TreeId treeId, NodeId parentId, size_t position, size_t featureIndex, double featureValue)
+    NodeId addSplitNode(TreeId treeId, NodeId parentId, size_t position, size_t featureIndex, double featureValue, int yes_if_missing = 0)
```

Comment:
I think, these `yes_if_missing`/`yesIfMissing` naming just confusing. Doesn't be worth rename it like `any_missing_values`/`if_missing`/`any_missing`?

Reply:
Thanks for your feedback!
The meaning of this parameter can be described as following: "In case of missing value take place, one should go to "yes" way in the split point". For this reason I guess, `yes_if_missing` is more clear naming than `any_missing/if_missing/any_missing_values`. May be we could try to find better replacement.

Reply:
Also, we're mixing `snake_case` and `camelCase` here. Please avoid that. Following your description. I would avoid putting details of the implementation in the variable name (what are the `yes` and `no` ways anyways? depends on our implementation!), and rather name it according to the expected effect. What about `splitIfValueIsMissing` or similar? And why is it an `int` and not a `bool` then?

Reply:
How is the `bool` vs. `int` problem solved. We could convert to `bool` right at the interface, so that at least all C++ code would be consistent.

I like `defaultLeft` !

Reply:
> How is the bool vs. int problem solved. We could convert to bool right at the interface, so that at least all C++ code would be consistent.

This values should be stored in `HomogenNumericTable`. As far as I see, one can't use `HomogenNumericTable<bool>`.